src/components/slider/slider-base.ts

```diff
@@ -544,8 +544,37 @@ export class IgcSliderBaseComponent extends LitElement {
     return groups;
   }
 
-  protected renderThumb(value: number, ariaLabel?: string, thumbId?: string) {
+  private handleFocus(ev: Event) {
+    this.activeThumb = ev.target as HTMLElement;
+    const thumbId = this.activeThumb?.id;
+    const thumbVal = this.activeThumb?.ariaValueNow;
+
+    const thumbs = this.shadowRoot?.querySelectorAll('div[part="thumb"]');
+    thumbs?.forEach((t) => {
+      if (t.id !== thumbId) {
+        this.activeThumb?.setAttribute(
+          'aria-valuetext',
+          `${Math.min(
+            parseInt(thumbVal!),
+            parseInt(t.ariaValueNow!)
+          )} - ${Math.max(parseInt(thumbVal!), parseInt(t.ariaValueNow!))}`
+        );
+      }
+    });
+  }
+
+  protected renderThumb(
+    value: number,
+    ariaLabel?: string,
+    thumbId?: string,
+    ariaValueText?: string
```

Comment:
A bit of nitpick but since all params are kind of internal to the component can you simplify the call signature to just the thumbId and get rest based on that?

Reply:
I could, but how would you pass the aria label and the valueText like we do in `renderThumbs()` in range-slider.ts?